VoiceController.php

```diff
@@ -1,48 +1,94 @@
-public function voice(Request $request){
+<?php
+
+function voice(Request $request)
+{
     $request->validate([
         'question_id'=>'required|int|exists:questions,id',
         'value'=>'required|boolean',
     ]);
+    
+    $AuthUser = auth()->user();
 
-    $question=Question::find($request->post('question_id'));
-    if (!$question)
-        return response()->json([
-            'status'=>404,
-            'message'=>'not found question ..'
-        ]);
-    if ($question->user_id==auth()->id())
+    $question = Question::findOrFail($request->question_id);
```

Comment:
@muhamedRadwan this is shorter but loses the original error message of "not found question .."